pkg/kubelet/kuberuntime/kuberuntime_container_windows.go

```diff
@@ -128,11 +128,6 @@ func (m *kubeGenericRuntimeManager) generateWindowsContainerConfig(container *v1
 	if username != "" {
 		wc.SecurityContext.RunAsUsername = username
 	}
-	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.WindowsGMSA) &&
```

Comment:
Why you remove the whole if statement? Shouldn't you remove only the `tilfeature.DefaultFeatureGate.Enabled(kubefeatures.WindowsGMSA)` part? @marosset , correct me if I am wrong

Reply:
Yes, you are correct. How i do to revert it in this PR? Should I open other PR?

Reply:
You can continue in this PR. Just keep editing files in this branch. And then squash your commits. See https://www.kubernetes.dev/docs/guide/github-workflow/#squash-commits